hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/action/rollback/AbstractMarkerBasedRollbackStrategy.java

```diff
@@ -90,6 +90,8 @@ protected HoodieRollbackStat undoAppend(String appendBaseFilePath, HoodieInstant
     String fileId = FSUtils.getFileIdFromFilePath(baseFilePathForAppend);
     String baseCommitTime = FSUtils.getCommitTime(baseFilePathForAppend.getName());
     String partitionPath = FSUtils.getRelativePartitionPath(new Path(basePath), new Path(basePath, appendBaseFilePath).getParent());
+    final Map<FileStatus, Long> writtenLogFileSizeMap = config.useFileListingMetadata()
```

Comment:
Hmmm. can we remove this flag as we discussed before? using the file listing flag here is very confusing and crosses a lot of layers. if you are worried about the new code breaking, then a catch block returning an empty map is good? Downside is it will make it hard to detect issues here.